### DISCUSSION TITLE
was there ever a key in Netbox titled "physical_ip4", or was this a custom field someone in my org created?

### DISCUSSION THREAD

**Discussion by farfigneugan** [netbox-community/netbox#11735] [Q&A] [answered]
The person who set up our production network automation pipeline left the org back in August. 
We're using Netbox as our source of truth, with device updates triggering a webhook to send to python where we collect the info and define our jinja filters, then store the generated configs in gitlab. 

 I'm working on some changes to the template to create a new bgp peer, but to avoid causing issues I'm trying to test outside of the production environment, so I'm mimicking the prod environment without actually storing in the develop branch repository in gitlab, potentially causing serious network issues. 

 The problem I'm running into is that (at least) one of the filters is trying to call a json key from the IP called "physical_ip4", but these aren't defined so it errors at that point. 
 
 I could have sworn I remember actually seeing a key in the API that had that name, but now I'm not actually finding it anywhere. It's possible it was simply a custom field he'd added to our Netbox, but I really thought it was there before. Can anyone remember ever seeing that before? 

This key is being referenced in an api call to /dcim/interfaces/x 

Note: I'm not confusing this with "primary_ip4", I know where that is in the api/dcim/devices.

**Comment by jeremystretch** [comment, marked as answer]
To my knowledge, we have never used the name `physical_ip4` anywhere in NetBox core. It's likely it was defined as a custom field in your deployment.
